apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -59,23 +60,36 @@ import useHeaders from "./useHeaders";
 import {
   convertFormValuesToApiCreateInput,
   convertFormValuesToApiUpdateInput,
-  getLocalizedSkillLevel,
   FormValues,
 } from "./utils";
 
+const getSkillLevelMessage = (
+  poolSkill: PoolSkill | undefined,
+  intl: IntlShape,
+): string => {
+  let skillLevel = "";
+  if (poolSkill?.requiredLevel && poolSkill.skill) {
+    skillLevel =
+      poolSkill.skill.category === SkillCategory.Technical
+        ? intl.formatMessage(getTechnicalSkillLevel(poolSkill.requiredLevel))
+        : intl.formatMessage(getBehaviouralSkillLevel(poolSkill.requiredLevel));
+  } else skillLevel = intl.formatMessage(commonMessages.notFound);
+
+  return skillLevel;
+};
```

Comment:
I dont think this is part of the scope, unless you really feel like it 😅 

But I think we have this pattern for both name and definition of skills levels a few places now. Would be nice if we could create a small helper so we don't need to keep repeating ourselves. Do you think I should create a ticket for that?

Reply:
Yeah, I think it would be a good idea so please go ahead. I think this logic will probably appear in more places so there will be further repeating,